### PR TITLE
S3t5 compare olddog newdog

### PR DESCRIPTION
**Pull request by lizsharp**
Trigger the getDogs function

### CODE REVIEW

File: src/components/CardContainer/CardContainer.js [hunk 1]
```diff
@@ -30,12 +30,26 @@ class CardContainer extends React.Component {
...
+    clickUpdateWin = () => {
+        //Matt work here
+        // document.getElementsByClassName("like-button").addEventListener("click", );
+        let dogs = [this.state.randomDog(), this.state.randomDog()]
+        while ((dogs[0]._id == this.state.dogs[0]._id || dogs[0]._id == this.state.dogs[1]._id)
+            && (dogs[1]._id == this.state.dogs[0]._id || dogs[1]._id == this.state.dogs[1]._id)) {
+            dogs = [this.state.randomDog(), this.state.randomDog()]
+        }
+        const state = {...this.state, dogs: dogs}
+        this.setState(state)
```
Comment (mallowmew) [inline src/components/CardContainer/CardContainer.js]: Maybe pull this out into its own function.

File: src/components/CardContainer/CardContainer.js [hunk 2]
```diff
@@ -30,12 +30,26 @@ class CardContainer extends React.Component {
...
-                const state = {...this.state, dogs: dogs}
+                const state = {...this.state, dogs: dogs, randomDog: random}
                 this.setState(state)
                 }
             )
     }
 
+    clickUpdateWin = () => {
+        //Matt work here
+        // document.getElementsByClassName("like-button").addEventListener("click", );
```
Comment (mallowmew) [inline src/components/CardContainer/CardContainer.js]: delete commented-out old code

File: src/components/CardContainer/CardContainer.js [hunk 3]
```diff
@@ -30,12 +30,26 @@ class CardContainer extends React.Component {
...
+        let dogs = [this.state.randomDog(), this.state.randomDog()]
+        while ((dogs[0]._id == this.state.dogs[0]._id || dogs[0]._id == this.state.dogs[1]._id)
+            && (dogs[1]._id == this.state.dogs[0]._id || dogs[1]._id == this.state.dogs[1]._id)) {
+            dogs = [this.state.randomDog(), this.state.randomDog()]
+        }
+        const state = {...this.state, dogs: dogs}
+        this.setState(state)
+        //Rachmann work here
+        //this.getDogs()
```
Comment (mallowmew) [inline src/components/CardContainer/CardContainer.js]: delete commented-out old code

File: src/components/CardContainer/CardContainer.js (hunk 1 above)
Comment (mallowmew) [reply src/components/CardContainer/CardContainer.js]: This doesn't block merge for me - could equally be done as a cleanup task, & it depends on how difficult to read the other code that's going in this function will be.

File: src/components/CardContainer/CardContainer.js (hunk 2 above)
Comment (lizsharp) [reply src/components/CardContainer/CardContainer.js]: Done.

**Pull request merged**